Configurations/unix-Makefile.tmpl

```diff
@@ -1127,6 +1127,17 @@ doc-nits: build_generated_pods
 md-nits:
 	mdl -s util/markdownlint.rb .
 
+GENERATED_FIPS_HEADERS = \
+     providers/common/include/prov/der_digests.h \
+     providers/common/include/prov/der_dsa.h \
+     providers/common/include/prov/der_ec.h \
+     providers/common/include/prov/der_ecx.h \
+     providers/common/include/prov/der_rsa.h \
+     providers/common/include/prov/der_sm2.h
+
+include-nits: build_generated $(GENERATED_FIPS_HEADERS)
```

Comment:
It's better to use `build_all_generated` than to ad *another manually maintained list of files*!

Reply:
The trade-off is that `build_all_generated` formats the manual pages, outputs a warning about `make depend and perl` and such.

Reply:
I'm fine with that trade-off.  Manually maintained lists are evil.

Reply:
I would prefer to see an auto-generated list that could be used here.  I don't see why the FIPS headers aren't handled the same as the other openssl headers.  Yes, someone might not configure with `enable-fips` but the incremental cost of running the script six times is approximately zero.

Reply:
I am trying to make an auto-generated list of the generated FIPS header files.  Lines like this, fail:
```
SOURCE[FIPSHEADERS] = $DER_DIGESTS_H
```
Fail.  As does this:
```
DEPENDS[FIPSHEADERS] = $DER_DIGESTS_H
``
Also fail.  Do I need make target or variable for `FIPSHEADERS` somewhere?